styles.css

```diff
@@ -433,19 +494,30 @@ svg {
   height: 4em;
 }
 
-.site-header__text:hover {
+.site-header__text,
+#link-to-top {
```

Comment:
Can we add a class to the link to top element, it can even be `class="link-to-top"`, and select on that here instead of an ID? IDs have a very large precedence, making them cumbersome to override, leading to an inheritance war ending quickly in `!important` soup.